aodndata/moorings/products_handler.py

```diff
@@ -206,6 +206,16 @@ def _get_old_product_files(self):
                                                                       product_url=product_url)
             )
 
+    def _handle_errors(self, errors):
```

Comment:
the function name is not the intent - you are just reporting and adding files to a list.
I would rename to "check_excluded_files" or alike

Reply:
renamed to `_log_excluded_files`